examples/rest/basic_fields.py

```diff
@@ -26,6 +26,7 @@ class BasicFieldsSerializer(serializers.ModelSerializer):
     date_field = fields.DateField()
     time_field = fields.TimeField()
     duration_field = fields.DurationField()
+    password_field = fields.CharField(password_field=True)
```

Comment:
Tu me malo skrbi, ker je v testih ena funkcija, ki nafutra BasicFields tabelo. Ali to še vse skupaj še vedno dela, ko dodaš to polje? Vsekakor se to password polje ne nafila za test.